src/main/resources/adapterdefinisjoner/k9-oppgavetyper-k9punsj.json

```diff
@@ -41,6 +41,11 @@
           "visPåOppgave": true,
           "påkrevd": false
         },
+        {
+          "id": "journalpostId",
+          "visPåOppgave": true,
+          "påkrevd": false
```

Comment:
Kan denne settes som påkrevd for punsj? Vi får vel alltid journalpostId der?

Reply:
Ja, hvis det fungerer å innføre et nytt felt uten å sette defaultverdi? Jeg fikk problemer med dette da jeg la til mottattDato.